packages/automerge-repo/src/DocHandle.ts

```diff
@@ -327,6 +328,19 @@ export class DocHandle<T> //
     })
   }
 
+  /** `setRemoteHeads` is called by the doc synchronizer
+   * @hidden
+   */
+  setRemoteHeads(peerId: PeerId, heads: A.Heads) {
```

Comment:
I don't love this API for reasons we've already discussed but I'll put up with it for now.